Sming/SmingCore/Network/TcpServer.cpp

```diff
@@ -85,7 +85,7 @@ void TcpServer::setTimeOut(uint16_t waitTimeOut)
 bool TcpServer::listen(int port)
 {
 	if (tcp == NULL)
-		tcp = tcp_new();
+		initialize(tcp_new());
```

Comment:
It happens here, on `connect`: https://github.com/gbl08ma/Sming/blob/4fd2fe5356fd6cfb6431bf85e0276cbdb5de7133/Sming/SmingCore/Network/TcpConnection.cpp#L45

...but I just realized that I missed the overload that takes an IP address. It should check if `tcp == NULL` there too and initialize a tcp_new if yes. I'll add the two missing lines...